### PR TITLE
Instructions to try aws based registration manually

### PR DESCRIPTION
**Pull request by jaswalkiranavtar**

### CODE REVIEW

File: content/en/blog/joining-hub-and-spoke-with-aws-auth-manually/index.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    export HUB_POLICY_NAME=hub
+    export SPOKE_POLICY_NAME=spoke
+    export SPOKE_OIDC_PROVIDER_ID=BE6D831CFA22823093D17E608EE0048C
+    export HUB_REGION=us-west-2
+    export SPOKE_REGION=us-west-2
+   ```
+
+2. Login to spoke AWS account and create IAM role, policy and tags on spoke IAM. Go to the folder containing this markdown file in this [repository](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io) and run following commands:
+   ```bash
+   sed -e "s/PROVIDER_ID/$SPOKE_OIDC_PROVIDER_ID/g" -e "s/ACCOUNT_ID/$SPOKE_ACCOUNT_ID/g" -e "s/REGION/$SPOKE_REGION/g" templates/Template-Spoke-Role-Trust-Policy.json > templates/Spoke-Role-Trust-Policy.json
```
Comment (mikeshng) [inline content/en/blog/joining-hub-and-spoke-with-aws-auth-manually/index.md:52]: I am not sure if this is the best place to hold the `templates` folder. 
Maybe https://github.com/open-cluster-management-io/ocm/tree/main/solutions makes more sense? @qiujian16 @xuezhaojun WDYT?

Comment (qiujian16) [reply content/en/blog/joining-hub-and-spoke-with-aws-auth-manually/index.md:52]: yes, I agree.

Comment (suvaanshkumar) [reply content/en/blog/joining-hub-and-spoke-with-aws-auth-manually/index.md:52]: Created [PR](https://github.com/open-cluster-management-io/ocm/pull/749/files)

**Review by mikeshng — CHANGES_REQUESTED** [review]

File: content/en/blog/joining-hub-and-spoke-with-aws-auth-manually/index.md (hunk 1 above)
Comment (jaswalkiranavtar) [reply content/en/blog/joining-hub-and-spoke-with-aws-auth-manually/index.md]: Moved

**Review by mikeshng — APPROVED** [review]
/approve

/lgtm

Thanks!

**Pull request merged**